server/modules/auth/auth.controller.ts

```
@@ -0,0 +1,372 @@
+      last_name,
+      email,
+      password, // Password will be hashed by the model's pre-save hook
+      role: role || "user",
+    });
+
+    // Send verification email
+    await authService.sendVerificationEmail(user);
+
+    // Remove sensitive data from response
+    const userWithoutPassword = user.toObject();
+    delete userWithoutPassword.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)